### PR TITLE
Sets up GitHub Workflows

### LINKED ISSUE #12: Create Workflow to Create DMG File

### PR DESCRIPTION
**Pull request by khurd21**
closes #12

### CODE REVIEW

File: autoresponder/responder.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 from .models import Configuration, SingleChat, GroupChat, DefaultSingleChat
 from imessage_reader.fetch_data import FetchData
 from imessage_reader.data_container import MessageData
-from emojipasta import Emojipasta
-
+from emoji import emojize
```
Comment (SilasStokes) [inline autoresponder/responder.py:4]: this is the wrong package lol

File: .github/workflows/create-dmg.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                pip3 install -r requirements.txt
+            - name: Make dmg
+              run: |
+                cp ./configs/example-config.json ./configs/config.json
+                ./make_dmg.sh
+            - name: Find DMG file
+              id: find_dmg
+              run: |
+                dmg_file=$(find dist/ -maxdepth 1 -type f -name "*.dmg")
+                echo "::set-output name=dmg_file::$dmg_file"
```
Comment (SilasStokes) [inline .github/workflows/create-dmg.yml:29]: interesting

File: autoresponder/responder.py (hunk 1 above)
Comment (khurd21) [reply autoresponder/responder.py:4]: lol oops.  What package was it?

File: .github/workflows/create-dmg.yml (hunk 2 above)
Comment (khurd21) [reply .github/workflows/create-dmg.yml:29]: I'm going to find another way to do it. This is one command that says is deprecated

File: autoresponder/responder.py (hunk 1 above)
Comment (khurd21) [reply autoresponder/responder.py:4]: Still emoji pasta?

**Comment by khurd21** [comment]
https://github.com/SilasStokes/pymessage_gpt_bot/actions/runs/8028375584

I got the warnings to go away and for an artifact to get uploaded. My last update here was to make sure that the workflow only kicks off on a merge into main. One annoying thing I noticed was that it was made on an intel chip mac. Maybe it is better this way for now as it lets the older Macs also use this dmg file.

**Comment by SilasStokes** [comment]
Looks great Kyle! Do we have to do a tagged commit on main to get the github action to fire or is it on every merge/commit?

**Comment by khurd21** [comment]
It runs on any commit / PR into main. Tagging would only be if you wanted to give people a specific release to download!

**Pull request merged**